gitops/github/ManifestGeneration.md

```diff
@@ -0,0 +1,163 @@
+# Guide: Manifest Generation Pipeline
```

Comment:
Curious to understand why this is simply a copy and paste from the gitops/azure-devops/. I think it's fine to reuse some of the context here, but it might be good to revise certain sections so that it caters to Github and not Azure DevOps.

For example, the first sentence explicitly states "This section describes how to configure Azure DevOps to be your CI/CD....". This could be confusing if a user navigates to this directory hoping to learn how to use GitHub but ends up reading about AzDO.

Reply:
yes, its a typo

Reply:
fixed